sim/deathknight/scourge_strike.go

```diff
@@ -76,8 +76,6 @@ func (dk *Deathknight) registerScourgeStrikeSpell() {
 					if dk.DiseasesAreActive(spellEffect.Target) {
 						dk.LastScourgeStrikeDamage = spellEffect.Damage
 						shadowDamageSpell.Cast(sim, spellEffect.Target)
-						shadowDamageSpell.SpellMetrics[spellEffect.Target.Index].Casts--
```

Comment:
so I removed this because it was making the average damage look weird... I am not sure the best way to handle this.

Reply:
Yeah i've noticed but without this the shadow part is even messier. Maybe we can just split them somehow? I know that if I use rank3 spell Id for the shadow part it shows up as a separate ability in the details with proper metrics but thats just wrong in my head